harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -610,7 +610,7 @@ def _compute_validation_metrics(self) -> workload.Metrics:
             pytorch._convert_metrics_to_numpy(self.context.reduce_metrics(for_training=False))
         )
 
-        if self.context.distributed.size > 1 and any(
+        if self.context.distributed.size > 1 and self.use_horovod and any(
```

Comment:
how would you get the correct metrics in all workers if you don't broadcast in the pytorch-native dtrain case?